docs/how-to/enforce-permissions/user-permissions.mdx

```diff
@@ -160,7 +160,40 @@ UserPermissions permissions = permit.getUserPermissions(
 );
 
 ```
-</TabItem>
 
+##  Get user permissions directly from opa
+
```

Comment:
add content here, something like
When having high load, it might be more performant to call the OPA engine directly from the SDK.
Note that to do that you'll also have to expose OPA port